tests/nnstreamer_repo/runTest.sh

```diff
@@ -31,8 +31,16 @@ else
 fi
 convertBMP2PNG
 
-# The first gst buffer at tensor_reposrc is dummy.
+# Fail Test : Negotiation Error (dimension)
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} multifilesrc location=testsequence_%1d.png index=0 caps=\"image/png, framerate=30/1\" ! pngdec ! tensor_converter ! tensor_reposink silent=false slot-index=0 tensor_reposrc silent=false slot-index=0 caps=\"other/tensor, dim1=3, dim2=100, dim3=100, dim4=1, type=uint8, framerate=30/1\" ! multifilesink location=testsequence01_%1d.log" F0 0 1 $PERFORMANCE
+
+# Fail Test : Negotiation Error (type)
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} multifilesrc location=testsequence_%1d.png index=0 caps=\"image/png, framerate=30/1\" ! pngdec ! tensor_converter ! tensor_reposink silent=false slot-index=0 tensor_reposrc silent=false slot-index=0 caps=\"other/tensor, dim1=3, dim2=100, dim3=100, dim4=1, type=float32, framerate=30/1\" ! multifilesink location=testsequence01_%1d.log" F1 0 1 $PERFORMANCE
```

Comment:
Fail Test (type)
change to valid dimension to test negotiation failure of tensor type (3:100:100:1 -> 3:16:16:1)

Reply:
Nice Catch!. Thanks.